lib/compare_with_wikidata.rb

```diff
@@ -21,22 +21,22 @@ class DiffOutputGenerator
 
     def initialize(mediawiki_site:, page_title:)
       @mediawiki_site = mediawiki_site
-      @page_title = page_title.gsub('_', ' ')
+      @page_title = page_title.tr('_', ' ')
     end
 
     def run!
       # Note that the comparison is lazily evaluated when 'comparison'
       # is evaluated from the binding in template rendering:
 
       always_overwrite = {
-        '/stats' => 'templates/stats.erb',
-        '/comparison' => 'templates/comparison.erb',
-        '/_default_header_template' => 'templates/header_template.erb',
-        '/_default_footer_template' => 'templates/footer_template.erb',
-        '/_default_row_added_template' => 'templates/row_added.erb',
-        '/_default_row_removed_template' => 'templates/row_removed.erb',
+        '/stats'                          => 'templates/stats.erb',
```

Comment:
Personally, I really dislike this default stylistic convention of Rubocop, which means what should be single line changes often involve changing the rest of the lines in the group, and produce larger diffs unless you're suppressing whitespace changes. (It's discouraged here http://mysociety.github.io/coding-standards.html#indentation-and-whitespace .)

Ideally, I'd like to disable that cop in the [everypolitician-data `.rubocop_base.yml`](https://raw.githubusercontent.com/everypolitician/everypolitician-data/master/.rubocop_base.yml) but perhaps that would cause too many knock-on changes, in which case I'd prefer to disable it here anyway.

Reply:
@mhl: I think there are at least a couple of different issues here:

This isn't actually a `rubocop`-wide default: it's an everypolitician-rubocop default setting. So I don't think this would be about _disabling_ the cop entirely, but choosing a different configuration for it.

However, as far as I can tell, the other [options for this cop](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/AlignHash) only allow you to configure how the *keys* are laid out, not the values — i.e. there is no other option that would prevent a layout like:

```
      always_overwrite = {
        '/stats' => 'templates/stats.erb',
        '/comparison' =>      'templates/comparison.erb',
        '/_default_header_template' =>      'templates/header_template.erb',
        '/_default_footer_template' => 'templates/footer_template.erb',
      }
```

Personally I actually _like_ the "table" layout, and perhaps because I do most of my reviewing via Github's web interface, which makes it quite clear that only whitespace has changed, I don't really have an issue with the diffs created:

![screen shot 2018-04-09 at 14 18 35](https://user-images.githubusercontent.com/57483/38499923-eca535d0-3c00-11e8-8065-6cc4172858c9.png)

If there were an option that forced a single space after the separator, for example, I'd probably switch to it here, despite my preference for the table layout. But where the options are "the values should be aligned" or "there's no checking whatsoever of the layout of values", I'm a bit more hesitant to simply turn checking off entirely. 

I'm also keen that this doesn't get bogged down here though, so I'm going to take the "we can merge this PR regardless" path for now, and we can always update the local `.rubocop.yml` when we have consensus.